architecture/additional_concepts/other_api_objects.adoc

```diff
@@ -117,20 +117,24 @@ metadata:
   selflink: "/osapi/v1/oAuthClients/openshift-web-console"
   resourceVersion: "1"
   creationTimestamp: "2015-01-01T01:01:01Z"
-respondWithChallenges: false <2>
-secret: "45e27750-a8aa-11e4-b2ea-3c970e4b7ffe" <3>
+  accessTokenMaxAgeSeconds: "172800" <2>
```

Comment:
don't make the default override ... if you want to include the field, set it to nil in the example, and document the meaning of `0` and non-zero values

Reply:
also, this is a number, not a string